docs/content/concepts/partitions-schedules-sensors/sensors.mdx

```diff
@@ -144,23 +142,19 @@ def my_asset_sensor(context: SensorEvaluationContext, asset_event: EventLogEntry
 
 Multi-asset sensors, which can trigger job executions based on multiple asset materialization event streams, can be handled using the <PyObject object="multi_asset_sensor" decorator/> decorator.
 
-In the body of the sensor, you have access to the materialization event records for each asset via the `context`. The context provides two methods for fetching the materialization event records:
-
-- <PyObject
-  object="MultiAssetSensorEvaluationContext"
-  method="latest_materialization_records_by_key"
-  />
-  :
+In the body of the sensor, you have access to the materialization event records for each asset via the `context`. The context provides methods for fetching the materialization event records, as well as methods to handle partitioned materializations:
 
-Returns a dictionary mapping the `AssetKey` for each monitored asset to the most recent materialization record. If there is no materialization event, the mapped value will be `None`
+| Method                                                                                                      | Description                                                                                                                                                                                                          |
```

Comment:
This is duplicative of the API reference, and I think we should just point people there to avoid the concept page getting massive. Thoughts?

Reply:
got it--I will remove this